apps/src/code-studio/components/progress/CourseProgress.jsx

```diff
@@ -97,7 +97,7 @@ const CourseProgress = React.createClass({
                   <CourseProgressRow
                     stage={stage}
                     key={stage.name}
-                    isFocusArea={focusAreaPositions.indexOf(count++) > -1}
+                    isFocusArea={focusAreaStageIds.includes(stage.id)}
```

Comment:
afaict, the only thing we use these for is to do lookups. Would we be better off storing this as a dictionary in our redux store, i.e. instead of `focusAreaStageIds = [123, 456]` we have `focusAreaStageIds = { 123: true, 456: true}`, at which point we can do `isFocusArea = !!focusAreaStageIds[stage.id]`

actually, probably an entirely unnecessary optimization given that this array will presumably be super small